build-eslint-command.js

```diff
@@ -0,0 +1,9 @@
+const shelljs = require(`shelljs`);
```

Comment:
The main reason for using shelljs here as opposed to execSync is that this file is being linted by eslint and fails via the no-sync enforcement. Is there a proper way to get around this or is using shelljs ok in this instance?